src/SprykerEco/Service/AkeneoPim/Dependencies/External/Api/Adapter/ReferenceEntity/ReferenceEntityApiAdapter.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Service\AkeneoPim\Dependencies\External\Api\Adapter\ReferenceEntity;
+
+use Akeneo\Pim\ApiClient\Pagination\ResourceCursorInterface;
+use Akeneo\PimEnterprise\ApiClient\AkeneoPimEnterpriseClientInterface;
+use SprykerEco\Service\AkeneoPim\Dependencies\External\Api\Wrapper\WrapperFactoryInterface;
+
+class ReferenceEntityApiAdapter implements ReferenceEntityApiAdapterInterface
+{
+    /**
+     * @var \Akeneo\PimEnterprise\ApiClient\AkeneoPimEnterpriseClientInterface
+     */
+    protected $akeneoPimClient;
+
+    /**
+     * @var \SprykerEco\Service\AkeneoPim\Dependencies\External\Api\Wrapper\WrapperFactoryInterface
+     */
+    protected $wrapperFactory;
+
+    /**
+     * @param \Akeneo\Pim\ApiClient\AkeneoPimClientInterface $akeneoPimClient
+     * @param \SprykerEco\Service\AkeneoPim\Dependencies\External\Api\Wrapper\WrapperFactoryInterface $wrapperFactory
+     */
+    public function __construct(AkeneoPimEnterpriseClientInterface $akeneoPimClient, WrapperFactoryInterface $wrapperFactory)
+    {
+        $this->akeneoPimClient = $akeneoPimClient;
+        $this->wrapperFactory = $wrapperFactory;
+    }
+
+    /**
+     * @param string $referenceEntityCode
+     *
+     * @return array
+     */
+    public function get(string $referenceEntityCode): array
```

Comment:
Have a question regarding this methods that return array. What exactly we have inside it? Mb it's possible to map it to transfer?

Reply:
Data array here. See the specification of the original SDK method.

    /**
     * Gets a single reference entity.
     *
     * @param string $referenceEntityCode Code of the reference entity
     *
     * @throws HttpException If the request failed.
     *
     * @return array
     */
    public function get(string $referenceEntityCode): array;